assets/js/translator.js

```diff
@@ -622,19 +622,30 @@ function populateTranslationList() {
         $('#srcLangSelect').append(
             $('<option></option>')
                 .prop('value', this)
-                .html((isVariant(this) ? '&nbsp;&nbsp;&nbsp;&nbsp;' : '') + getLangByCode(this))
+                .html((isVariant(this) ? '    ' : '') + getLangByCode(this))
         );
     });
     $.each(dstLangs, function () {
         $('#dstLangSelect').append(
             $('<option></option>')
                 .prop('value', this)
-                .html((isVariant(this) ? '&nbsp;&nbsp;&nbsp;&nbsp;' : '') + getLangByCode(this))
+                .html((isVariant(this) ? '    ' : '') + getLangByCode(this))
         );
     });
 
     $('#srcLangSelect').val(curSrcLang);
+    var currentSrcLangSelect = $('#srcLangSelect option[value="' + $('#srcLangSelect').val() + '"]');
+    // Check if language begins with &nbsp;
+    if(currentSrcLangSelect.text().charAt(0) === ' ') {
+        // If it does, temporarily remove the four spaces
+        currentSrcLangSelect.text(currentSrcLangSelect.text().substring(4)); // eslint-disable-line no-magic-numbers
```

Comment:
Just replace the non-breaking spaces now. Also, safe to remove the comments; the code's intention is now clear :) Also, `.startsWith(` is nicer than the `charAt`.